File: 1-clase/C#/clase1.cs
// using System;

// class Program {
//     static void Main() {
//         // Datos de un empleado
//         string nombre = "Juan Pérez";
//         double salarioBase = 2500.50;
//         double bonificacion = 300.75;

//         double salarioTotal = salarioBase + bonificacion;

//         Console.WriteLine("Empleado: " + nombre);
//         Console.WriteLine("Salario Base: " + salarioBase);
//         Console.WriteLine("Bonificación: " + bonificacion);
//         Console.WriteLine("Salario Total: " + salarioTotal);
//     }
// }











// using System;

// class Program {
//     static void Main() {
//         // Verificar mayoría de edad
//         int edad = 20;  // Cambia este valor para probar
//         bool esMayor = edad >= 18;

//         Console.WriteLine("Edad: " + edad);
//         Console.WriteLine("¿Es mayor de edad? " + (esMayor ? "Sí" : "No"));
//     }
// }



// using System;

// class Program {
//     static void Main() {
//         // Cálculo del IMC
//         double peso = 70;  // kg
//         double altura = 1.75;  // m

//         double imc = peso / Math.Pow(altura, 2);
//         string estado = imc < 18.5 ? "Bajo peso" :
//                         imc < 25 ? "Peso normal" :
//                         imc < 30 ? "Sobrepeso" : "Obesidad";

//         Console.WriteLine("IMC: " + imc.ToString("0.00"));
//         Console.WriteLine("Estado: " + estado);
//     }
// }



using System;

class Program {
    static void Main() {
        // Calculadora de descuentos en C#
        Console.Write("Ingrese el nombre del producto: ");
        string producto = Console.ReadLine();

        Console.Write("Ingrese el precio original: ");
        double precio = Convert.ToDouble(Console.ReadLine());

        Console.Write("Ingrese el porcentaje de descuento: ");
        double descuento = Convert.ToDouble(Console.ReadLine());

        if (descuento >= 0 && descuento <= 100) {
            double precioFinal = precio - (precio * descuento / 100);
            Console.WriteLine($"✅ Producto: {producto}");
            Console.WriteLine($"💲 Precio original: ${precio:F2}");
            Console.WriteLine($"🔹 Descuento aplicado: {descuento}%");
            Console.WriteLine($"💰 Precio final: ${precioFinal:F2}");
        } else {
            Console.WriteLine("❌ Error: El porcentaje de descuento debe estar entre 0 y 100.");
        }
    }
}

File: 1-clase/python/clase1.py
# nombre = "Juan Perez"
# salario_base=2500.50
# bonificacion = 300.75

# salario_total= salario_base+ bonificacion

# print("Salario Total",salario_total)

# edad =20
# es_mayor = edad >=18

# print("Edad ",edad)
# print("Es Mayor de edad?", "Si" if es_mayor else "No")

# Cálculo del IMC
# peso = 70  # kg
# altura = 1.75  # m

# imc = peso / (altura ** 2)
# estado = "Bajo peso" if imc < 18.5 else "Peso normal" if imc < 25 else "Sobrepeso" if imc < 30 else "Obesidad"

# print(f"IMC: {imc:.2f}")
# print("Estado:", estado)

# Calculadora de descuentos en Python
producto = input("Ingrese el nombre del producto: ")
precio = float(input("Ingrese el precio original: "))
descuento = float(input("Ingrese el porcentaje de descuento: "))

if 0 <= descuento <= 100:
    precio_final = precio - (precio * descuento / 100)
    print(f"✅ Producto: {producto}")
    print(f"💲 Precio original: ${precio:.2f}")
    print(f"🔹 Descuento aplicado: {descuento}%")
    print(f"💰 Precio final: ${precio_final:.2f}")
else:
    print("❌ Error: El porcentaje de descuento debe estar entre 0 y 100.")
